module/chunks/chunkVerifier_test.go

```diff
@@ -346,3 +396,34 @@ func (vm *vmMock) Run(ctx fvm.Context, proc fvm.Procedure, led state.View, progr
 
 	return nil
 }
+
+type vmSystemOkMock struct{}
+
+func (vm *vmSystemOkMock) Run(ctx fvm.Context, proc fvm.Procedure, led state.View, programs *programs.Programs) error {
+	tx, ok := proc.(*fvm.TransactionProcedure)
+	if !ok {
+		return fmt.Errorf("invokable is not a transaction")
+	}
+
+	tx.ServiceEvents = []flow.Event{epochSetupEvent}
```

Comment:
It wasn't clear to me that this was why this mock was "OK" and the other one was "Bad". Could rename to be more clear eg:
```go
tx.ServiceEvents = expectedServiceEvents
```

or add a comment